eacl/record.go

```diff
@@ -91,12 +99,16 @@ func (r *Record) SetAction(action Action) {
 }
 
 // AddRecordTarget adds single Target to the Record.
+// Deprecated: use [Record.SetTargets] instead.
```

Comment:
> Each time you're using getters/setters you're doing something wrong.

no idea where this statement comes from

> Why removing these?

it’s more common to work with slices. In some cases it will save from reallocs

Reply:
This just means that slices can be exposed directly, without get/set wrappers.

Reply:
until generics for structure fields appear, the interface will be more powerful w/o any costs